website/docs/installation/evaluation.md

```diff
@@ -4,30 +4,49 @@ sidebar_position: 1
 
 # Evaluation
 
-We provide an [**evaluation** image](https://ghcr.io/ferretdb/ferretdb-eval:2)
+We provide an [**evaluation** image](https://ghcr.io/ferretdb/ferretdb-documentdb-eval:2)
```

Comment:
The URL was valid; this change makes it invalid. It was just copied from the old private repo without checking. That's bad.

Please do your own review before requesting a re-review.